### DISCUSSION TITLE
All I see is an empty screen that says "plate". What am I missing?

### DISCUSSION THREAD

**Discussion by spetryjohnson** [HASwitchPlate/openHASP#635] [Q&A] [answered]
Long-time HA user, first time OpenHASP user here.

I have flashed two different nightly binaries (7.0-rc9 and rc10) onto my SenseCAP Indicator D1. Both operations went smoothly, but when the device boots it tells me its IP address, and then all I see is an empty screen with the word "plate" at the upper left.

![PXL_20240211_122538627](https://github.com/HASwitchPlate/openHASP/assets/275393/12da544d-4778-4712-a290-0e19df537654)

The docs all contain screenshots of settings panels and the like, which I cannot figure out how to access. Am I doing something totally stupid?

_(I tried finding older versions of the sensecap firmware, but most of the assets for the historical build actions have expired. I don't suppose anyone has a version of the 6.x firmware they could give me access to?)_

Thanks!

**Comment by fvanroie** [comment, marked as answer]
The firmware does not contain any screens. You need to [design a page](https://www.openhasp.com/0.7.0/design/pages/) (or copy someone else's design). This is typically done by [editing or uploading](https://www.openhasp.com/0.7.0/design/file-editor/) a `pages.jsonl` file.

**Reply by spetryjohnson** [reply]
Yeah, I knew that, what I was missing was that I needed to go to the device's IP address in my browser to do the configuration.

The screenshots made it look like the settings panels were being displayed on the device itself.

Do you take PRs on documentation? I'd be happy to add a FAQ or something to help the next idiot that comes along with the same question...

**Reply by fvanroie** [reply]
Yes, the documentation could use an external set of eyes, as most is written from a developer point of view.
